src/platform/site-wide/announcements/selectors.js

```diff
@@ -1,20 +1,25 @@
+// Node modules.
+import moment from 'moment-timezone';
+// Relative imports.
 import _config from './config';
-import moment from 'moment';
 
-function isExpiredAnnouncement(announcement) {
+// Checks if the announcement has expired.
+const isExpiredAnnouncement = announcement => {
   if (!announcement.expiresAt) return true;
 
-  const expirationDate = moment(announcement.expiresAt);
-  const isExpired = moment().isSameOrAfter(expirationDate);
+  const expiresAtDate = moment.tz(announcement.expiresAt, 'America/New_York');
+  const hasExpired = moment()
+    .tz('America/New_York')
+    .isSameOrAfter(expiresAtDate);
 
-  return !isExpired;
-}
+  return !hasExpired;
```

Comment:
The `VeteransDayProclamation` is expired, but any changes to the expiration logic make a little nervous it'll resurrect itself lol. Can we disable or delete the `VeteransDayProclamation` to be safe?

Reply:
Removed! :)